### PR TITLE
Rank mentors by number of mentees

### PR DESCRIPTION
**Pull request by tsleal29**

### CODE REVIEW

File: app/controllers/mentors_controller.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def index
     # TODO: order by ascending number of pairs (available mentors should be first)
     # TODO: count only active S&H mentorships
     # TODO: preload mentorship pairs to view (avoid n+1)
-    @mentors = User.where(id: users_with_mentor.select(:mentor_id))
+    @mentors = User.where(id: users_with_mentor.select(:mentor_id)).sort_by{|user| user.mentees.count}.reverse
```
Comment (fwuensche) [inline app/controllers/mentors_controller.rb:10]: We don't have a CI yet, but you have a rubocop issue here. 
Can you check it locally ?

Comment (tsleal29) [reply app/controllers/mentors_controller.rb]: > We don't have a CI yet, but you have a rubocop issue here.
> Can you check it locally ?

Done

File: app/models/user.rb [hunk 2]
```diff
@@ -4,8 +4,9 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :rememberable, :confirmable, :lockable, :timeoutable, :recoverable, :trackable and :validatable
   devise :database_authenticatable, :registerable, :omniauthable
-
+
```
Comment (fwuensche) [inline app/models/user.rb:7]: set your text editor (vscode, of course) to automatically trim trailing whitespaces: 

```
  "files.trimTrailingWhitespace": true,
```

File: app/controllers/mentors_controller.rb [hunk 3]
```diff
@@ -7,7 +7,7 @@ def index
     # TODO: order by ascending number of pairs (available mentors should be first)
     # TODO: count only active S&H mentorships
     # TODO: preload mentorship pairs to view (avoid n+1)
-    @mentors = User.where(id: users_with_mentor.select(:mentor_id))
+    @mentors = User.where(id: users_with_mentor.select(:mentor_id)).sort_by { |user| user.mentees.count }.reverse
```
Comment (fwuensche) [inline app/controllers/mentors_controller.rb:10]: great. it's a n+1 query tho, but we can fix it later when it becomes a problem.

File: app/models/user.rb (hunk 2 above)
Comment (tsleal29) [reply app/models/user.rb:7]: Thanks for the tip

Added these two:
    "files.insertFinalNewline": true,
    "files.trimTrailingWhitespace": true

Comment (fwuensche) [reply app/models/user.rb:7]: yes. can you remove it from here too?

Comment (tsleal29) [reply app/models/user.rb]: Sure. Done

**Pull request merged**